wikimedia/wikipedia-ios
# Update Notification Center type images for consistent padding/margins

**Phabricator:** https://phabricator.wikimedia.org/T288658 (in part)

### Notes
Previously, these leading images were mixed between manually exported PDFs and SF Symbols. This led to inconsistent image padding/margins in the leading image container. This PR updates those images to all be exported PDF assets with an intrinsic size of 32x32. This way, we can precisely control the expected margin/padding metrics in the PDF assets themselves.

### Test Steps
1. Confirm the new leading images render in app and generally match the padding/margin metrics in Figma. I found it easiest to just change [this type](https://github.com/wikimedia/wikipedia-ios/blob/364bb378cbafeb71049e8db25ee9b510ae144a19/Wikipedia/Code/NotificationsCenterCell.swift#L297) to each of the cases in `NotificationsCenterCellStyle`'s `leadingImage(_ displayState:...)` to confirm they rendered in app.
